web/docs/setup/build.md

```diff
@@ -29,7 +29,7 @@ dependencies and versions.
 |✓|[CMake](https://cmake.org)|>= 3.19|Cross-platform tool for building, testing and packaging software.|
 |✓|[CAF](https://github.com/actor-framework/actor-framework)|>= 0.17.6|Implementation of the actor model in C++. (Bundled as submodule.)|
 |✓|[FlatBuffers](https://google.github.io/flatbuffers/)|>= 1.12.0|Memory-efficient cross-platform serialization library.|
-|✓|[Apache Arrow](https://arrow.apache.org)|>= 8.0.0|Required for in-memory data representation. Must be built with Compute, Zstd and Parquet enabled.|
+|✓|[Apache Arrow](https://arrow.apache.org)|>= 10.0.0|Required for in-memory data representation. Must be built with Compute, Zstd and Parquet enabled.|
```

Comment:
Can this doc change be reverted or are we "officially" supporting Arrow >=10.0.0 now?

Reply:
Good observation! I reverted it now.